micromasters/settings.py

```diff
@@ -553,3 +554,10 @@ def show_toolbar(request):
         return not excluded
 
     DEBUG_TOOLBAR_CONFIG = {"SHOW_TOOLBAR_CALLBACK": show_toolbar, }
+
+# Exams SFTP
+EXAMS_SFTP_HOST = get_var('EXAMS_SFTP_HOST', 'localhost')
+EXAMS_SFTP_PORT = get_var('EXAMS_SFTP_PORT', '22')
+EXAMS_SFTP_USERNAME = get_var('EXAMS_SFTP_USERNAME', '')
```

Comment:
If we don't have a good default for this can we default it to `None` instead and raise an exception where it's used?